examples/chart/teleport-cluster/charts/teleport-operator/values.yaml

```
@@ -120,6 +120,13 @@ rbac:
   # where the user deploying the operator is not allowed to edit RBAC resources.
   create: true
 
+# extraEnv(list)
+# Extra environment to be configured on the pod
```

The comment syntax is `name(type) -- explanation`, else the documentation generator won't pick it up.

```suggestion
extraEnv(list) -- contains extra environment to be configured on the pod.
```